driver-core/src/main/com/mongodb/internal/operation/AggregateOperationImpl.java

```diff
@@ -202,16 +202,16 @@ private CommandCreator getCommandCreator(final SessionContext sessionContext) {
             @Override
             public BsonDocument create(final ServerDescription serverDescription, final ConnectionDescription connectionDescription) {
                 validateReadConcernAndCollation(connectionDescription, sessionContext.getReadConcern(), collation);
-                return getCommand(connectionDescription, sessionContext);
+                return getCommand(sessionContext);
             }
         };
     }
 
-    private BsonDocument getCommand(final ConnectionDescription description, final SessionContext sessionContext) {
+    BsonDocument getCommand(final SessionContext sessionContext) {
```

Comment:
Just removing unused parameters so that it's easier to call the method from explain helpers